tests/bsim/bluetooth/host/iso/cis/src/cis_peripheral.c

```diff
@@ -159,13 +156,15 @@ static void adv_connect(void)
 static void test_main(void)
 {
 	init();
-	adv_connect();
-	WAIT_FOR_FLAG_SET(flag_iso_connected);
-	WAIT_FOR_FLAG_SET(flag_data_received);
-	WAIT_FOR_FLAG_UNSET(flag_iso_connected);
-	WAIT_FOR_FLAG_UNSET(flag_connected);
 
-	PASS("Test passed\n");
+	for (;;) {
```

Comment:
It seems `for(;;)` is used more in embedded code than in PC apps :)

Reply:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/2520413/aa786727-d800-4df1-a0eb-4186a5a4c86d)

![image](https://github.com/zephyrproject-rtos/zephyr/assets/2520413/083c6f4b-8336-4639-9548-9a690ece0410)

![image](https://github.com/zephyrproject-rtos/zephyr/assets/2520413/375ff9a5-b179-4dde-8ee5-40c8251fe959)

It may be used more in embedded than on PCs, but definitely not the common solution in Zephyr ;)

Reply:
Yea it was just a drive-by remark, not disagreeing.